Code/src/megadesk.cpp

```diff
@@ -956,25 +986,27 @@ byte recvInitPacket(byte array[])
 
 void initAndReadEEPROM(bool force)
 {
-  int signature = eepromGet16(EEPROM_LOCATION_SIG);
+  int signature = eepromGet16(EEPROM_SIG_SLOT);
 
   if ((signature != MAGIC_SIG) || force)
   {
     for (unsigned int index = 0; index < EEPROM.length(); index++)
       EEPROM.write(index, 0);
     // Store signature value
-    eepromPut16(EEPROM_LOCATION_SIG, MAGIC_SIG);
+    eepromPut16(EEPROM_SIG_SLOT, MAGIC_SIG);
 
-    #ifdef MINMAX
-    // reset max/min height
-    eepromPut16(MIN_HEIGHT_SLOT, DANGER_MIN_HEIGHT);
-    eepromPut16(MAX_HEIGHT_SLOT, DANGER_MAX_HEIGHT);
-    #endif
   }
   #ifdef MINMAX
+  // reset max/min height
   minHeight = eepromGet16(MIN_HEIGHT_SLOT);
   maxHeight = eepromGet16(MAX_HEIGHT_SLOT);
+  // check if 0 and fix/beep ...
+  if (minHeight == 0) toggleMinHeight();
+  if (maxHeight == 0) toggleMaxHeight();
```

Comment:
I think reading 0 from eeprom for min/maxHeight is probably bad, so that's avoided here.